src/main/kotlin/net/irisshaders/lilybot/extensions/moderation/Report.kt

```diff
@@ -368,7 +353,7 @@ class Report : Extension() {
 
 	inner class ManualReportArgs : Arguments() {
 		val message by string {
-			name = "message-link"
+			name = "messageLink"
```

Comment:
I don't think capital letters are allowed in this regex. Will need testing

Reply:
They are, see config args. They just don't actually display in Discord. I changed it like this for internal consistency.